src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -304,9 +305,9 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
 
             try {
                 if (isset($this->fileMap[$id])) {
-                    return $this->load($this->fileMap[$id]);
+                    return self::IGNORE_ON_UNINITIALIZED_REFERENCE === $invalidBehavior ? null : $this->load($this->fileMap[$id]);
```

Comment:
can't the value `self::IGNORE_ON_UNINITIALIZED_REFERENCE === $invalidBehavior` be stored in a variable ?

Reply:
it can, but it's better as is to me.

Reply:
Np then ;)